ci/playbooks/edpm_build_images/edpm_image_builder.yml

```diff
@@ -3,6 +3,19 @@
   ansible.builtin.import_playbook: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/ci-framework/ci_framework/playbooks/01-bootstrap.yml"
 - hosts: all
   tasks:
+    - name: Read hash from delorean.repo.md5 file
+      tags:
+        - edpm_build_img
+      slurp:
+        path: "{{ cifmw_basedir }}/artifacts/repositories/delorean.repo.md5"
+      register: file_content
+
+    - name: Set fact for cifmw_edpm_build_images_tag var
+      tags:
+        - edpm_build_img
+      set_fact:
+        cifmw_edpm_build_images_tag: "{{ file_content['content'] | b64decode | string }}"
```

Comment:
Need to trim extra line \n

https://logserver.rdoproject.org/47/347/3249e5364faf087abfa716e9065c5994f8205b28/github-check/periodic-edpm-build-push-images-centos-9-antelope/1d22522/job-output.txt

2023-07-07 01:20:53.778083 | controller | TASK [edpm_build_images : Push edpm-hardened-uefi image to registry name={{ item }}, push_args={'dest': '{{ cifmw_edpm_build_images_push_registry }}/{{ cifmw_edpm_build_images_push_registry_namespace }}'}, tag={{ cifmw_edpm_build_images_tag }}, pull=False, push=True] ***
2023-07-07 01:20:53.778093 | controller | Friday 07 July 2023  01:20:53 -0400 (0:00:21.443)       0:15:36.101 ***********
2023-07-07 01:20:55.347045 | controller | failed: [localhost] (item=edpm-hardened-uefi) => {"ansible_loop_var": "item", "changed": false, "item": "edpm-hardened-uefi", "msg": "Failed to find image edpm-hardened-uefi:44a88d6e0610f1523e468f623da8e163\n locally, image pull set to False"}
2023-07-07 01:20:55.347102 | controller | failed: [localhost] (item=ironic-python-agent) => {"ansible_loop_var": "item", "changed": false, "item": "ironic-python-agent", "msg": "Failed to find image ironic-python-agent:44a88d6e0610f1523e468f623da8e163\n locally, image pull set to False"}

Reply:
done